static/app/components/passwordStrength.tsx

```diff
@@ -32,16 +31,28 @@ type Props = {
  * NOTE: Do not import this component synchronously. The zxcvbn library is
  * relatively large. This component should be loaded async as a split chunk.
  */
-export function PasswordStrength({
-  value,
-  labels = ['Very Weak', 'Very Weak', 'Weak', 'Strong', 'Very Strong'],
-  colors = [theme.red300, theme.red300, theme.yellow300, theme.green300, theme.green300],
-}: Props) {
-  if (value === '') {
+export function PasswordStrength(props: Props) {
```

Comment:
this component might live outside the theme? can't remember

Reply:
https://github.com/getsentry/sentry/blob/master/static/app/bootstrap/processInitQueue.tsx#L49-L53

Reply:
damn, let me try and run this to see if we can access it in another way

Reply:
Will revert this component, but we might get away here by wrapping it inside a theme provider if the config value from the scope above contains the theme preference. I need to look into it a bit more